src/cbmc/cbmc_parse_options.cpp

```diff
@@ -977,6 +983,9 @@ void cbmc_parse_optionst::help()
     " --xml-ui                     use XML-formatted output\n"
     " --xml-interface              bi-directional XML interface\n"
     " --json-ui                    use JSON-formatted output\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
+    " --validate-goto-model        enable additional well-formedness checks on the\n"
+    "                              goto program\n"
     HELP_GOTO_TRACE
```

Comment:
I'd not add this flag to the help. It's for CBMC developers only, and is a distraction for users.